### ISSUE TITLE
Cannot find Lunii_Box

### ISSUE THREAD

**Issue by campafap** [kairoh/studio#12]
**Describe the bug**
I cannot understand

Here the message from Terminal : 

An error occurred while handling device plug event
java.util.concurrent.CompletionException: studio.core.v1.utils.exception.StoryTellerException: Could not locate device partition
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:314)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:319)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1803)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1792)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:295)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
Caused by: studio.core.v1.utils.exception.StoryTellerException: Could not locate device partition
	at studio.driver.fs.FsStoryTellerAsyncDriver.lambda$new$2(FsStoryTellerAsyncDriver.java:91)
	at studio.driver.LibUsbDetectionHelper.lambda$registerCallback$1(LibUsbDetectionHelper.java:133)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1800)
	... 6 more

any idea ?

**Expected behavior**
How to put history in the box ? I don't see it...

**Desktop and environment (please complete the following information):**
 - OS: MacOS 13.0.1
 - Browser [chrome and safari]
 - Java version: [Java 8 mise à jour 351 (last one)]
 - STUdio application version [e.g. 4.02]

**Comment by campafap** [comment]
Thanks a lot for any help

**Comment by gilles6** [comment]
Same error here on **macOS Ventura 13.1**, `mount` shows:
```
$ mount
/dev/disk1s1s1 on / (apfs, sealed, local, read-only, journaled)
devfs on /dev (devfs, local, nobrowse)
/dev/disk1s3 on /System/Volumes/Preboot (apfs, local, journaled, nobrowse)
/dev/disk1s5 on /System/Volumes/VM (apfs, local, noexec, journaled, noatime, nobrowse)
/dev/disk1s6 on /System/Volumes/Update (apfs, local, journaled, nobrowse)
/dev/disk1s2 on /System/Volumes/Data (apfs, local, journaled, nobrowse)
map auto_home on /System/Volumes/Data/home (autofs, automounted, nobrowse)
msdos://disk2/LUNII on /Volumes/LUNII (lifs, local, nodev, nosuid, noowners, noatime)
```

On **macOS Monterey 12.8** where it works, `mount` shows:
```
$ mount
/dev/disk1s1s1 on / (apfs, sealed, local, read-only, journaled)
devfs on /dev (devfs, local, nobrowse)
/dev/disk1s5 on /System/Volumes/VM (apfs, local, noexec, journaled, noatime, nobrowse)
/dev/disk1s3 on /System/Volumes/Preboot (apfs, local, journaled, nobrowse)
/dev/disk1s6 on /System/Volumes/Update (apfs, local, journaled, nobrowse)
/dev/disk1s2 on /System/Volumes/Data (apfs, local, journaled, nobrowse)
map auto_home on /System/Volumes/Data/home (autofs, automounted, nobrowse)
/dev/disk2 on /Volumes/LUNII (msdos, local, nodev, nosuid, noowners)
```

**Comment by Seph29** [comment]
I confirm, I compiled my own version with modification #13  and it works

**Comment by campafap** [comment]
@Seph29 I can't edit the file... Could you help me ? I need a 0.4.3 to download :( or at least the fixed file

**Comment by campafap** [comment]
I can't compile :(

**Comment by Seph29** [comment]
@campafap Mail me

**Comment by dthib** [comment]
Hi @Seph29 , why is your patch behind v0.4.2?
Thanks

**Comment by kairoh** [comment]
Should be fixed in 1.0.0
